core/src/main/java/org/kiwix/kiwixmobile/core/page/adapter/Page.kt

```diff
@@ -25,8 +25,8 @@ interface PageRelated {
 interface Page : PageRelated {
   val zimFilePath: String?
   val zimId: String
-  val url: String
-  val title: String
+  val url: String?
+  val title: String?
```

Comment:
`url` and `title` should be non-nullable.

Reply:
done